modules/ding_ddbasic/modules/ding_ddbasic_opening_hours/ding_ddbasic_opening_hours.pages.inc

```diff
@@ -21,6 +21,13 @@ function ding_ddbasic_opening_hours_week_callback($node, $from, $to, $categories
       'categories' => $categories,
     ));
   }
+  else {
+    $tmp = theme('ding_ddbasic_opening_hours_week', array(
+      'node' => $node,
+      'timespan' => array($from, $to),
+      'categories' => $categories,
+    ));
+  }

```

Comment:
It can be safely removed from the `if` so we don't repeat, as it repeats in both `if` and in `else`.